cSploit/src/org/csploit/android/core/System.java

```diff
@@ -145,10 +146,10 @@
 
   private final static LinkedList<SettingReceiver> mSettingReceivers = new LinkedList<SettingReceiver>();
 
-  public static void init(Context context) throws Exception{
+  public static void init(Context context, String iface) throws Exception{
```

Comment:
Now, almost yes. The network conf (wifiInfo + target network + target gateway + target device) was originally initialized here: https://github.com/cSploit/android/blob/master/cSploit/src/org/csploit/android/core/System.java#L190

So I used the iface parameter to initialize the network with the selected interface.

Later, I decided to remove that portion of code because it was the same than the one used in reloadNetworkMapping(), and use this method from displayNetworkInterfaces()
https://github.com/cSploit/android/blob/master/cSploit/src/org/csploit/android/core/System.java#L340

Now it'll only indicate if it's the first time we launch the app (the unique case where iface can be null), because as it's called from this method (MainActivity.init(), it won't be null during the app lifetime (or it shouldn't):
https://github.com/gainan/android/blob/b470506ace528a7ba146f5b6b736e38e96578837/cSploit/src/org/csploit/android/MainActivity.java#L479